simple-calculator/mycalculator.ts

```diff
@@ -0,0 +1,33 @@
+function calculator(x:number) {
+let firstoperator: any;
+let secondoperator: any;
+let resultt: any;
+
+firstoperator=document.getElementById("1");
```

Comment:
`const firstOperator = +document.getElementById("number1").value;`

Did you try defining, initializing and casting at the same time ?
It will make your next logic more readable.

`result.value = firstOperator + secondOperator;` is more readable than `resultt.value=1*firstoperator.value+1*secondoperator.value;`